### DISCUSSION TITLE
Improve navigation & components for screen readers

### DISCUSSION THREAD

**Discussion by brandonkelly** [craftcms/cms#8407] [Ideas]
WCAG reference: http://www.w3.org/WAI/WCAG20/quickref/

Test as a screen reader with Chrome Vox: https://chrome.google.com/webstore/detail/chromevox/kgejglhpjiefppelpmljglcjbhoiplfn

Accessible drag'n'drop: https://dev.opera.com/articles/accessible-drag-and-drop/

**Comment by nternetinspired** [comment]
Just a heads-up on a testing tool that I've found very useful recently for a11y testing against WCAG: 

https://open-indy.github.io/Koa11y/
